### PR TITLE
Adds a(nother) slick cowboy hat!

### PR DESCRIPTION
**Pull request by etherware-novice**
# About The Pull Request
Adds a sweet new cowboy hat to the clothesmate to make it more accessible to the crew to do cool showdowns

## Why It's Good For The Game
More roleplay power being able to buy a hat if you want

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

cowhat_vend

cowhat_floor

cowhat_wear

</details>

## Changelog

:cl:
:add: added the cowboy hat(?) item
:tweak: added the cowboy hat(?) to the clothemate
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
You need to update your Master branch. This PR modifies catgloves.

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -89,6 +89,7 @@
 		            /obj/item/clothing/accessory/waistcoat = 1,
 					/obj/item/clothing/head/that = 1,
 		            /obj/item/clothing/head/fedora = 1,
+					/obj/item/clothing/head/cowboydiscount = 3,  // for low budget "town aint big enough" scenes!
```
Comment (nednaZ) [inline code/modules/vending/clothesmate.dm:92]: The comment should say that it is a monkestation edit.

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -44,6 +44,12 @@
 	icon_state = "mailman"
 	desc = "<i>'Right-on-time'</i> mail service head wear."
 
+/obj/item/clothing/head/cowboydiscount
```
Comment (nednaZ) [inline code/modules/clothing/head/misc.dm:47]: Use snake_case for readability.
```suggestion
/obj/item/clothing/head/cowboy_discount
```

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -44,6 +44,12 @@
...
+	name = "cowboy hat(?)"
```
Comment (nednaZ) [inline code/modules/clothing/head/misc.dm:48]: Just give it a regular name
```suggestion
	name = "cowboy hat"
```

**Review by nednaZ — CHANGES_REQUESTED** [review]
Don't put the icon changes in the base folders.
Use the monkestation module file for it.

**Comment by etherware-novice** [comment]
> Don't put the icon changes in the base folders.
> 
> Use the monkestation module file for it.

im having issues getting the human worn sprite working properly, which file do i put it in?

**Comment by Zonespace27** [comment]
> > Don't put the icon changes in the base folders.
> > Use the monkestation module file for it.
> 
> im having issues getting the human worn sprite working properly, which file do i put it in?

you can VV it in-game to see what icon file it uses for worn objects

**Comment by etherware-novice** [comment]
whats vving it?
> > > Don't put the icon changes in the base folders.
> 
> > > Use the monkestation module file for it.
> 
> > 
> 
> > im having issues getting the human worn sprite working properly, which file do i put it in?
> 
> 
> 
> you can VV it in-game to see what icon file it uses for worn objects

**Comment by Zonespace27** [comment]
> what's VVing it?

View Variables

File: monkestation/code/modules/clothing/head/misc.dm [hunk 4]
```diff
@@ -113,6 +113,15 @@
 	user.update_inv_head() //Regenerate the wearer's head appearance so that they have real-time hat updates.
 
 
+/obj/item/clothing/head/cowboy_discount
```
Comment (nednaZ) [inline monkestation/code/modules/clothing/head/misc.dm:116]: Wrong location for this.
I'd suggest checking the file structure for similar hats.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Your files are a mess here.
You're modifying the original files, as well as the new files.
Somehow you're reducing the file size on the inhand files, which means you've removed something.

Quality of the sprite seems pretty poor, the item sprite itself is incredibly oversized.

**Comment by etherware-novice** [comment]
you know what i give up i give up

**Pull request closed without merging**